src/plugins/dashboard/public/dashboard_listing/hooks/use_dashboard_listing_table.tsx

```diff
@@ -42,6 +42,7 @@ const toTableListViewSavedObject = (hit: DashboardItem): DashboardSavedObjectUse
     type: 'dashboard',
     id: hit.id,
     updatedAt: hit.updatedAt!,
+    namespaces: hit.namespaces ?? [],
```

Comment:
I think `namespaces` should always be present on the saved object, so we could also add an assertion operator rather than coalescing to an empty array. WDYT?